src/main/java/core/basesyntax/service/RegistrationServiceImpl.java

```diff
@@ -1,11 +1,64 @@
 package core.basesyntax.service;
 
+import core.basesyntax.dao.StorageDao;
 import core.basesyntax.model.User;
 
 public class RegistrationServiceImpl implements RegistrationService {
+    private static final int USER_MINIMUM_PASSWORD_LENGTH = 6;
+    private static final int USER_MINIMUM_AGE = 18;
+    private final StorageDao storageDao;
+
+    public RegistrationServiceImpl(StorageDao storageDao) {
+        this.storageDao = storageDao;
+    }
 
     @Override
     public User register(User user) {
-        return null;
+        if (!validateUser(user)) {
+            return null;
```

Comment:
I think this is unnecessary checking. If your user wouldn't pass your validation, you would throw an exception

Reply:
I think so, we mustn't return null in register method